### PR TITLE
User controller

### PR DESCRIPTION
**Pull request by ymclapp**

### CODE REVIEW

File: DaytaCare/Models/Identity/ApplicationUser.cs [hunk 1]
```diff
@@ -10,10 +10,10 @@ namespace DaytaCare.Models.Identity
 {
   public class ApplicationUser : IdentityUser
   {
-    [Required]
+    //[Required]
     public string FirstName {get; set;}
 
-    [Required]
+    //[Required]
```
Comment (iwhitmor) [inline DaytaCare/Models/Identity/ApplicationUser.cs:16]: Line 13 and 16 we can do in the User DTO (I think Keith said this in the slack chat)

File: DaytaCare/Controllers/UsersController.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    [HttpPost("Register")]
+    public async Task<IActionResult> Register(RegisterData data)
+    {
+      var user  = await userService.Register(data, this.ModelState);
+      if (user == null)
+        return BadRequest(new ValidationProblemDetails(ModelState)
+          );
+      return Ok();
```
Comment (iwhitmor) [inline DaytaCare/Controllers/UsersController.cs]: I think this might need to pass in the user
like:
return Ok(user);

**Review by iwhitmor — COMMENTED** [review]
Looks good. I left a couple of comments!!

File: DaytaCare/Controllers/UsersController.cs (hunk 2 above)
Comment (ymclapp) [reply DaytaCare/Controllers/UsersController.cs]: Checked notes and agree.  I added user.

File: DaytaCare/Models/Identity/ApplicationUser.cs (hunk 1 above)
Comment (ymclapp) [reply DaytaCare/Models/Identity/ApplicationUser.cs:16]: Done!

**Review by iwhitmor — APPROVED** [review]

**Pull request merged**